example/android/app/src/main/AndroidManifest.xml

```diff
@@ -31,9 +31,8 @@
                  until Flutter renders its first frame. It can be removed if
                  there is no splash screen (such as the default splash screen
                  defined in @style/LaunchTheme). -->
-            <meta-data
-                android:name="io.flutter.app.android.SplashScreenUntilFirstFrame"
-                android:value="true" />
+            <meta-data android:name="flutterEmbedding" android:value="2"/>
```

Comment:
I've read on flutter documentation this meta tag should be under application tag not activity.